### DISCUSSION TITLE
False "Mind map file was changed by another program. Reload?" alert

### DISCUSSION THREAD

**Discussion by halfbeing** [freeplane/freeplane#628] [Help]
I have just started using Freeplane on my Mac.

I frequently get a popup saying "Mind map file was changed by another program. Reload?", even though the mind map is not being used by any other program. In other words, it is a false alert.

I know I have the option to tick "Don't ask me again.", but I do plan to access my mind maps from multiple devices in the future, so this alert will be a useful one to have to prevent me accidentally losing work. How can I stop the false alerts from popping up without losing the valid alerts?

**Comment by macmarrum** [comment]
The alert is displayed when Freeplane detects for the mindmap file that its last modification time, reported by the file system, has changed. This could be caused by e.g. a synchronization agent, as mentioned in 
- https://github.com/freeplane/freeplane/discussions/484
